### DISCUSSION TITLE
File Based Licensing For Bootscore?

### DISCUSSION THREAD

**Discussion by kreincke** [bootscore/bootscore#364] [General]
Bootscore is licensed under the MIT license (https://github.com/bootscore/bootscore/blob/main/LICENSE), but the single files (templates, theme.js, and *.css) do not contain a license file header. Using an SPDX identifier would allow to add only a few signs for making clear that this file is covered by the general licensing statement: Spdx-License-Identifier: MIT (ttps://spdx.org/licenses/ )

What does the team think about adding such a file based licensing statement

**Comment by crftwrk** [comment]
I don't see any benefit in this other than that it complicates things. License here is exactly done the way all other do, for example https://github.com/twbs/bootstrap/blob/main/LICENSE.

However, if you have experience with spdx.org/licenses/ and think that this will improve the project, you're always welcome to create a PR.

**Comment by kreincke** [comment]
jeep, I am very familiar with this topic. It's my job. I know that many open source projects follow the method 'Put a license into the repo, that's enough'. But that's not sufficient. Formally you have to have a license and a licensing statement which links the license to your work. Your README.md contains such a statement. But you should also have a statement how many parts of the work is covered by this licensing statement.

During the last years SPDX became a standard to this (https://spdx.dev/wp-content/uploads/sites/41/2020/04/using_spdx_license_list_short_identifiers.pdf) I prefer to modify / work on files that have such a specific abbreviated licensing statement. Call it paranoid, but it's better for the future, I can minimize upcoming discussions.
Thanks for the invitation to add the identifier by myself and send a respective PR. I will do so during the next hour.

And of course, I will add the SPDX-ID only to those files, which I can identify as your work (you have the copyright) - hence: I won't add it into the bootstrap files and not into files with only call other files (themes.js)

small things matter ;-) best regards KR
